### PR TITLE
Cleura branding updates

### LINKED ISSUE #2: Need themeing for our brand
This is simply an issue to remember that we miss theme-ing, and site names.

### PR DESCRIPTION
**Pull request by fghaas**
With the move from City Network to Cleura, update the branding:

* Update the color palette.
* Add the Cleura logo and favicon.
* Update City Network and City Cloud references to Cleura and Cleura Cloud.

Also, add a facility to deploy rendered content to GitHub pages (so that this can go live on the target site, via a CNAME update).

Fixes #2.

### CODE REVIEW

**Comment by evrardjp** [comment]
Doing it asap.

File: theme/assets/stylesheets/cleura-palette.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[data-md-color-scheme="cleura"] {
+    --md-primary-fg-color: #FF6600;
```
Comment (evrardjp) [inline theme/assets/stylesheets/cleura-palette.css:2]: I didn't double check the values here. But the rest does make sense. Thanks for adding a palette scheme.

**Review by evrardjp — COMMENTED** [review]
It looks very nice.

Is there a way to merge some authors on the site? e.g. there are two Florian Haas, where it's obviously the same person. It's just because it's two different contribution email addresses.

**Comment by evrardjp** [comment]
Can you update the top PR comment, to add "Closes: #2" ?

**Comment by fghaas** [comment]
> Is there a way to merge some authors on the site?  e.g. there are two Florian Haas, where it's obviously the same person. It's just because it's two different contribution email addresses (https://timvink.github.io/mkdocs-git-authors-plugin/mailmap.html ? ).

Good suggestion! See df10a4c5ffedf1ea2da907005b9d9db8cd5d6802

**Comment by fghaas** [comment]
> Can you update the top PR text, to add "Closes: #2" ?

I did, though you adding this in a comment already had the same effect. GitHub is a little weird that way.

**Review by evrardjp — APPROVED** [review]
LGTM now

**Comment by fghaas** [comment]
@evrardjp Can you please refrain from merging PRs from the GitHub web interface (or `gh`)? GitHub doesn't ever do fast-forward merges that way (absolutely NFI why), so if you merge like this it forces me to rebase a PR like #14, which is an unnecessary workflow disruption.

Just approve the PRs, and I'll do a local `git merge --ff-only` and then push.

**Comment by evrardjp** [comment]
If you want us to remove the merge commit, it's pretty simple, and can be done in the default policy, is that what you want?

**Comment by fghaas** [comment]
The rebase merge doesn't solve the problem of having to _also_ rebasing any dependent PRs. 

I'll be happy to stand corrected, but to the best of my knowledge GitHub doesn't ever do fast-forward merges from the web UI or `gh`, even if the PR branch totally would allow it.

**Comment by evrardjp** [comment]
it doesn't automatically do it, sadly, but it offers the option to do so, with a click. Generally you associate that with a bot doing that click.

**Pull request merged**